packages/dds/matrix/test/matrix.spec.ts

```diff
@@ -668,39 +668,6 @@ describe("Matrix", () => {
                     ["B", "A", 2, 3]
                 ]);
             });
-
```

Comment:
The following commented-out test case is redundant.  (It's something I created temporarily when debugging long ago...)